app/package.json

```diff
@@ -9,14 +9,14 @@
     "develop": "REACT_APP_USE_SAMPLE_DATA=true yarn start",
     "build": "react-scripts build",
     "postbuild": "echo '# Keep directory in git.' > build/.gitkeep",
-    "test": "react-scripts test --env=jest-environment-jsdom",
+    "test": "react-scripts test --env=jest-environment-jsdom --transformIgnorePatterns \"node_modules/(?!d3)/\"",
```

Comment:
Preventing jest from running automated tests on our d3 dependencies? :thinking: 

Why would jest be running in `node_modules` for d3 specifically but not other dependencies?

Reply:
> Preventing jest from running automated tests on our d3 dependencies? 🤔

This doesn't excluded d3 from our tests. This just tells jest/babel to transpile the JS code in the d3 package. The latest versions of d3 are now published as an ESM module which jest doesn't natively support. Without doing this, the tests will fail due to babel not understanding the `export` keyword found in the d3 code. There are varying solutions to this problem, but this one I've added was the simplest fix that worked.

![image](https://github.com/lightninglabs/lightning-terminal/assets/1356600/5e5c7b1e-4001-4961-b9bd-084f2894757e)